includes/components/class-user.php

```diff
@@ -59,6 +59,7 @@ public function __construct( $args = [] ) {
 
 			// Alter templates.
 			add_filter( 'hivepress/v1/templates/site_footer_block', [ $this, 'alter_site_footer_block' ] );
+			add_filter( 'hivepress/v1/templates/user_email_verify_page/blocks', [ $this, 'alter_user_email_verify_page' ], 10, 2 );
```

Comment:
It's not a good practice to add global conditions to templates that define the content, this will not allow users to customize the template in HivePress/Templates (since it's not possible to set conditions in the block editor, and the template either shows the form or the message). Maybe a separate template is required to keep all the re-sending logic and specific message?